torch/csrc/autograd/custom_function.cpp

```diff
@@ -109,10 +114,38 @@ variable_list _wrap_outputs(const variable_list &input_vars,
     }
     set_history(var, i, is_input, is_modified, is_differentiable);
 
+    // For deprecation cycle. Can be removed after 1.6. In the case where we detected a view
+    // in no grad mode during the forward, only warn the user (do not change the flag if we
+    // return and input that is a view as is).
+    // See NOTE [ View + Inplace detection ] for why we replace everything by a warning.
```

Comment:
Do you know if any of the torchvision ops using this are affected by this? Shouldn't be too many users here.

Reply:
None one them use `.mark_dirty()` so, `is_modified` is always false here.
And after checking them, they don't ever make views (extract data_ptr and write to the result directly).